samples/PubSubSamples/RedisFunctionsDemo/WriteAroundSample.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Microsoft.Azure.WebJobs.Extensions.Redis.Samples.Models;
+using Microsoft.Extensions.Logging;
+using StackExchange.Redis;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Redis.Samples
+{
+    public static class WriteAroundSample
+    {
+        public const string localhostSetting = "redisLocalhost";
+        public const string cosmosDbConnectionSetting = "CosmosDBConnection";
+
+        private static readonly IDatabaseAsync s_redisDb =
+            ConnectionMultiplexer.ConnectAsync("<cache-name>.redis.cache.windows.net:6380,password=<access-key>,ssl=True,abortConnect=False,tiebreaker=").Result.GetDatabase();
```

Comment:
Using something like `ConnectionMultiplexer.ConnectAsync(configuration: ConfigurationManager.AppSettings["CacheConnection"].ToString()).Result.GetDatabase();` should get the connection from local.settings.json and that way you can avoid having this ever happening again

Reply:
Might actually be even better to use something like `Environment.GetEnvironmentVariable("key")` https://learn.microsoft.com/en-us/azure/azure-functions/functions-dotnet-class-library?tabs=v4%2Ccmd#environment-variables

Reply:
The latest commit uses `Environment.GetEnvironmentVariable("key")`